src/dnfile/base.py

```diff
@@ -441,21 +495,6 @@ class MDTableIndex(object):
     row_index: int
     row: MDTableRow
 
-    _table_class: Type[ClrMetaDataTable]
-
-    # TODO: figure out how and when to recursively resolve refs, and detect cycles
-
-    def __init__(self, value, tables_list: List[ClrMetaDataTable]):
-        self.row_index = value
-        for t in tables_list:
-            if isinstance(t, self._table_class):
-                self.table = t
-                if self.row_index > t.num_rows:
-                    # TODO error/warn
-                    self.row = None
-                    return
-                self.row = t.rows[self.row_index - 1]
-
```

Comment:
this code was unused. a more specific subclass is provided below as `MDTableIndexRef` that probably fills its place.

Reply:
agree